translations/it-IT.js

```diff
@@ -1,5 +1,13 @@
 module.exports = {
   I: 'io',
+  contexts: {
+    Feature: 'Caratteristica',
+    Scenario: 'lo_scenario',
+    Before: 'Prima',
+    After: 'Dopo',
+    BeforeSuite: 'Prima_della_suite',
+    AfterSuite: 'Dopo_la_suite',
+  },
   actions: {
     amOutsideAngularApp: 'sono_fuori_dalla_applicazione_angular',
     amInsideAngularApp: 'sono_dentro_la_applicazione_angular',
```

Comment:
would be nice if you consider changing them elsewhere 👍

Reply:
Sorry, Did not understand that properly. Can you please elaborate? :)

Reply:
I meant the same `CamelCase` changing. But yeah, depends on how it benefits.

Reply:
Changing this now will break the backward compatibility : (

Reply:
that would be `3.0` then, breaking things are tough! ^^